### DISCUSSION TITLE
a couple of omissions

### DISCUSSION THREAD

**Discussion by r4shpro** [Try/OpenGothic#475] [General]
Greetings Try,

it's a beautiful work you have done! Thank you for doing all this! I have noticed a couple of bugs that none has mentioned yet.

1) texture bug with the fire mage Daron at the marketplace in Khorinis, model of his robe casts an unusual peak, like texture/model artifact (checked the last version on Mac and Windows)
2) the player control remains a bit off the vanilla game. It's impossible to strife to a side whilst the previous animation is ongoing. When that animation ends you need to press movement to one or the other side, otherwise the player does not initiate any movement (if they are pressed, for instance, when the attack is ongoing) it was not the case with vanilla. Thus, a melee fight feels vastly different.

Thank you

**Comment by Try** [comment]
Hi, @r4shpro !

I've put some fixes to melee in d9b2b5aa commit - strafe should work now.

As for Daron, I'm not sure what "unusual peak", you talking about, here how it look on my side:
![изображение](https://github.com/Try/OpenGothic/assets/122894/3b5f9732-ec8d-4028-8a7e-e62d81411289)

**Reply by r4shpro** [reply]
I meant this one. I tested on Macbook pro m1, and old asus g551 with gtx860m. The peak even casts a proper shadow lol
<img width="601" alt="Screenshot 2023-07-01 at 23 39 37" src="https://github.com/Try/OpenGothic/assets/83694995/7ee44329-3086-4044-8194-78b8975137c2">

<img width="855" alt="Screenshot 2023-07-01 at 23 44 14" src="https://github.com/Try/OpenGothic/assets/83694995/b00de2cc-24ae-4d1d-8069-7ba4490f7195">
